util/DigitalOceanMarketplace/marketplace-image.json

```diff
@@ -0,0 +1,86 @@
+{
+  "variables": {
+    "do_token": "{{env `DIGITALOCEAN_TOKEN`}}",
+    "image_name": "bitwarden-20-04-snapshot-{{timestamp}}",
+    "apt_packages_1": "fail2ban ca-certificates curl gnupg lsb-release",
+    "apt_packages_2": "docker-ce docker-ce-cli containerd.io",
+    "application_name": "Bitwarden",
+    "application_version": "1.47.1"
```

Comment:
Since this is a self-host, do we want to continue with the server versioning or switch to the self-host script versioning? Will bumping the version from 1.31.* to 1.1.0 be a problem?

Reply:
I would say using the `self-host` script versioning would be very confusing in this scenario.  I think continuing with the `server` version would be best as it's closest to an application version.